tests/data/Cell/ConvertFormulaToA1FromR1C1Absolute.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+return [
+    // Basic arithmetic
+    ['=D3+F7+G4+C6+5', '=R3C4+R7C6+R4C7+R6C3+5'],
+    ['=D3-F7-G4-C6-5', '=R3C4-R7C6-R4C7-R6C3-5'],
+    ['=D3*F7*G4*C6*5', '=R3C4*R7C6*R4C7*R6C3*5'],
+    ['=D3/F7/G4/C6/5', '=R3C4/R7C6/R4C7/R6C3/5'],
+    // Formulas
+    ['=SUM(E1:E5)', '=SUM(R1C5:R5C5)'],
```

Comment:
Just as a useful little technique for your data provider array.... using associative indexes for each test can make it easier to recognise which tests are failing, and also has benefits if you report tests using additional phpunit tools
e.g.
```
return [
    'Sum with relative range' => ['=SUM(E1:E5)', '=SUM(R1C5:R5C5)'],
    'Sum with relative range and additional value' => ['=SUM(E1:E5, D5)', '=SUM(R1C5:R5C5, R5C4)'],
];
```
When a test does fail, it's reported by the name that you've given it, rather than simply it's offset number, which can make it a lot easier to identify in large data providers.

And not every test case needs to be named, so the array can be a mix of enumerated and associative

Reply:
Ah yes, I completely forgot about this. I've added in proper names for each data set.